cpp/algoritmins/sort/Order.cpp

```diff
@@ -175,6 +175,40 @@ void odd_even_sort(std::vector<T> &inputs, int p, int r) {
     }
   }
 }
+
+
+template <typename T>
+void count_sort(std::vector<T> &inputs, int p, int r, int exp){
+  int output[r]; 
+  int i, count[10] = { 0 };
+  for (i = p; i < r; i++)
+    count[(inputs[i] / exp) % 10]++;
+  for (i = 1; i < 10; i++)
+    count[i] += count[i - 1];
+  for (i = r - 1; i >= p; i--) {
+        output[count[(inputs[i] / exp) % 10] - 1] = inputs[i];
+        count[(inputs[i] / exp) % 10]--;
+    }
+  for (i = p; i < r; i++)
+        inputs[i] = output[i];
+}
+
+template <typename T>
+T getMax(std::vector<T> inputs, int p, int r){
+  auto mx=inputs[p];
+  for(int i=p+1; i<r; i++){
+    mx=max(mx,inputs[i]);
+  }
+  return mx;
+}
+
+template <typename T>
+void radix_sort(std::vector<T> &inputs, int p, int r){
+  auto mx = getMax<T>(inputs,p,r);
+  for (int exp = 1; mx / exp > 0; exp *= 10)
```

Comment:
```suggestion
  for (ssize_t exp = 1; mx / exp > 0; exp *= 10)
```

int is always good, also code in the Linux Kernel use int sometime, but in this library we stay safe and we use ssize_t